doc/release/1.0.0-notes.rst

```diff
@@ -75,6 +75,9 @@ has been changed to True.
 
 `scipy.special.gammaln` does not accept complex arguments anymore.
 
+The method `var` for the Dirichlet distribution of `scipy.stats.multivariate`
```

Comment:
nitpick: ``var`` needs double backticks. single ones mean a link, and this link won't resolve. double backticks are for formatting as code